doc/user_guide/create_new_sale_channel.rst

```diff
@@ -0,0 +1,69 @@
+New Sale Channel
+=================
+
+This module helps you understand how to create a new sale channel.
+
+New Module 
+-----------
+
+On the top, click on the ``Create a new record`` button.
+
+.. figure:: images/23.png
+
+In the window that pops up, you can see a number of fields.
+These are the fields on New Record Window:
+
+* **Name**: The name of your new sales channel. This field is manadatory.
+* **Code**: Optional Field, contains unique code generated during sale. 
+* **Company**: This is mandatory field to fill. It has the name of the company.
+* **Address**: Address where your company is located. This is optional field.
+* **Source**: How your channel will be handled. BY default it is set to Manual
+ entry.
+
+
+.. figure:: images/24.png
+
+Fill all required details and then click on the tab which say ``Order Defaults``.
```

Comment:
Just put another subheading called 'Order Defaults' instead of this redundant para